yt/yt/server/node/exec_node/volume_manager.cpp

```diff
@@ -1302,7 +1302,8 @@ class TLayerLocation
             {"place", PlacePath_}
         };
 
-        if (options.EnableDiskQuota && options.HasRootFSQuota) {
+        // NB: Root volume quota is independent from sandbox quota but enforces the same limits.
+        if (options.EnableRootVolumeDiskQuota) {
             volumeProperties["user"] = ToString(options.UserId);
```

Comment:
BTW in porto volume limit is implemented as project-quota which have nothing to do with "user".